tests/structuredheadings/formatcombo.js

```diff
@@ -37,13 +33,9 @@
 
     "if previous heading is autonumber, the new heading is autonumbered": function () {
       var bot = this.editorBot;
-      var ed = this.editor;
       bot.setHtmlWithSelection("<h1 class=\"autonumber autonumber-0\">bar</h1><p>^foo</p>");
-      var formatCombo = ed.ui.get(comboName);
-      assert.areSame(CKEDITOR.TRISTATE_OFF, formatCombo._.state, "check state OFF");
```

Comment:
assertions on the combo state seem to be prone to random timing issues and flakiness now that firefox is actually used for travis